package.json

```diff
@@ -1,23 +1,22 @@
 {
-  "name": "shkr",
-  "version": "0.0.1",
-  "private": true,
+  "name": "react-native-shake-to-report",
+  "version": "1.0.0",
+  "description": "Shake to report an issue component",
+  "main": "index.js",
+  "keywords": [
+    "Shake",
+    "Report"
+  ],
+  "homepage": "https://github.com/algotech/react-native-shake-to-report",
+  "author": "Algotech",
+  "license": "ISC",
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint ."
   },
-  "dependencies": {
-    "@react-navigation/native": "6.0.12",
```

Comment:
This one is not present in the list of "peerDependencies". Is it ok?

Reply:
Same thing with prop-types.

Reply:
Yes, the library doesn't use the navigation dependency anymore. I also added the prop-types to peerDependencies